src/test/java/org/mitre/synthea/modules/EncounterModuleTest.java

```diff
@@ -0,0 +1,55 @@
+package org.mitre.synthea.modules;
+
+import org.junit.Before;
+import org.junit.Test;
+
+import org.mitre.synthea.engine.Event;
+import org.mitre.synthea.helpers.Utilities;
+import org.mitre.synthea.world.agents.Person;
+import org.mitre.synthea.world.agents.Provider;
+import org.mitre.synthea.world.concepts.HealthRecord;
+import org.mitre.synthea.world.concepts.HealthRecord.Code;
+import org.mitre.synthea.world.concepts.HealthRecord.Encounter;
+import org.mockito.Mockito;
+
+import java.io.IOException;
+import java.util.Calendar;
+import java.util.LinkedList;
+
+import static org.junit.Assert.assertEquals;
+
+public class EncounterModuleTest {
+
+    private Person person;
+    private long time;
+
+    @Before
+    public void setup() throws IOException {
+        person = new Person(0L);
+
+        person.history = new LinkedList<>();
+
+        long birthTime = time - Utilities.convertTime("years", 35);
+        person.attributes.put(Person.BIRTHDATE, birthTime);
+        person.events.create(birthTime, Event.BIRTH, "Generator.run", true);
+
+        time = System.currentTimeMillis();
+    }
+
+    @Test
+    public void testEncounterWellness() {
+        Encounter encounter = person.record.encounterStart(time,
+                HealthRecord.EncounterType.WELLNESS.toString());
+        assertEquals("WELLNESS", encounter.type);
+        long timeArray[] = new long[]{ encounter.start, encounter.stop };
+        for(long l: timeArray){
+            Calendar c = Calendar.getInstance();
+            c.setTimeInMillis(l);
+            // Insure start/stop are a weekday
+            assertEquals(c.get(Calendar.DAY_OF_WEEK) != Calendar.SATURDAY, true);
```

Comment:
Will this fail if the test is run on a Saturday?

Reply:
Well, that might be a nice thing to test. Let me hardcode a date into the person that'll set a weekend.